Repositories/UserV1Repository.cs

```diff
@@ -25,49 +24,13 @@ public UserV1 GetUserById(int userId)
     {
         return _context.Users.SingleOrDefault(c => c.UserId == userId);
     }
-    
-    //Original UpdateUser method
-    // public UserV1 UpdateUser(UserV1 newUser)
-    // {
-    //     var originalUser = _context!.Users.Find(newUser.UserId);
-
-    //     if (originalUser != null)
-    //     {
-    //         // //Hash password if password is entered.
-    //         if (!string.IsNullOrEmpty(newUser.Password))
-    //         originalUser.Password = bcrypt.HashPassword(newUser.Password);
-            
-            // originalUser.UserId = newUser.UserId;
-            // originalUser.Username = newUser.Username;
-            // originalUser.Email = newUser.Email;
-            // originalUser.FirstName = newUser.FirstName;
-            // originalUser.LastName = newUser.LastName;
-            // originalUser.Location = newUser.Location;
-            // originalUser.MyPlaces = newUser.MyPlaces;
-    //         _context.SaveChanges();
-    //     }
-    //     return originalUser;
-    // }
 
     public void UpdateUser(int userId, UpdateRequest editUser)
     {
         var originalUser = GetUserById(userId);
-       
-        // validate (Not using in Version 1)
-        // if (model.Email != user.Email && _context.Users.Any(x => x.Email == model.Email))
-        //     throw new AppException("User with the email '" + model.Email + "' already exists");
-
-        // hash password if it was entered
-        if (!string.IsNullOrEmpty(editUser.Password))
-            // Where is the rest of the if statement {} code? 
-
-            originalUser.Password = bcrypt.HashPassword(editUser.Password);
 
         // copy model to user and save
 
```

Comment:
Add if statement around this code:
if (originalUser != null) { ...code}

Reply:
See my comment above.  It's impossible to change the password here, so hashing is not needed.